.github/workflows/ci-build.yml

```diff
@@ -114,7 +114,7 @@ jobs:
     - name: Setup go
       uses: actions/setup-go@v1
       with:
-        go-version: 1.15
+        go-version: 1.17
```

Comment:
Oof, good catch! It's a shame that dependabot doesn't update this (but, well, it can only do so much). I filed #1085 so we can document dependency updated like this that are outside of depdendabot's remit.